proposals/0000-swiftpm-script-support.md

```diff
@@ -123,12 +125,26 @@ SwiftPM will utilize the changes to the compiler front-end to access packages re
 
 Once package dependencies are known, SwiftPM will use it's existing logic for resolving those dependencies and outputting a ".resolved" file to ensure future invocations are reproducible.
 
-The name of the resolved file will be `script_name.resolved`.
+The name of the resolved file will be `script_name.resolved`. By default, the resolve file will be located under `~/.swiftpm/scripts` but can be specified to another path if desired.
+
+#### Updating package dependencies
+
+Dependencies of scripts can be updated to their latest version by executing `swift package update <script-name>`. 
+
+
+#### Converting a script to a Package
+
+SwiftPM will have new support for transforming a script to a package by extracting the information specified in `@package(...)` declarations.  
 
 #### Built products
 
 Products built from scripts will be located in common per-user location `~/.swiftpm/scripts/...`
 
+#### Shared local dependency cache
+
+Dependencies will be stored and managed by a local dependency cache which will allow scripts to avoid over-fetching
```

Comment:
This needs more discussion in the proposal. We should probably say something that communicate these points:

- It's quite common to use similar set of dependencies across many packages and scripts.
- Given that, shared dependency cache for dependency's sources is important for both packages and scripts to avoid performing unnecessary fetching operations.
- This proposal will make it very easy to spin up Swift scripts that share a lot of dependencies so it's important to implement the shared dependency cache in parallel with this proposal.